modules/recommended-etcd-practices.adhoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
```

Comment:
@chaitanyaenr  I just noticed that the file needs to be renamed .adoc (not .adhoc).

Reply:
Fixed it, thanks.